backend/app/models/evaluation_task.py

```diff
@@ -0,0 +1,64 @@
+import enum
+from uuid import UUID
+from typing import Optional, List
+
+from sqlalchemy import Text, JSON
+
+from sqlmodel import (
+    Field,
+    Column,
+    String,
+    Relationship as SQLRelationship,
+)
+
+from app.models.base import UpdatableBaseModel
+
+
+class EvaluationStatus(str, enum.Enum):
+    NOT_START = "not_start"
+    EVALUATING = "evaluating"
+    DONE = "done"
+    ERROR = "error"
+
+
+class EvaluationTask(UpdatableBaseModel, table=True):
+    id: Optional[int] = Field(default=None, primary_key=True)
+    name: str = Field(max_length=255)
+
+    user_id: UUID = Field(foreign_key="users.id", nullable=True)
+    user: "User" = SQLRelationship(
+        sa_relationship_kwargs={
+            "lazy": "joined",
+            "primaryjoin": "EvaluationTask.user_id == User.id",
+        },
+    )
+
+    dataset_id: int = Field(nullable=True)
+
+    evaluation_items: List["EvaluationItem"] = SQLRelationship(back_populates="evaluation_task")
+
+    __tablename__ = "evaluation_tasks"
+
+
+class EvaluationItem(UpdatableBaseModel, table=True):
```

Comment:
```suggestion
class EvaluationTaskItem(UpdatableBaseModel, table=True):
```
Would this be more reasonable?

Reply:
Sure, changed. Affected almost all files, PTAL again. @Mini256